chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
+#include<iostream>
+
+int Chess2::perform(str Move) {
```

Comment:
Can we declare a dataclass with Move consisting of

class Move{ int x1, int y1, int x2, int y2}